ipatests/test_xmlrpc/test_krbtpolicy.py

```diff
@@ -116,6 +116,35 @@ class test_krbtpolicy(Declarative):
             ),
         ),
 
+        dict(
+            desc='Reset global policy',
+            command=(
+                'krbtpolicy_reset', [], {}
+            ),
+            expected=dict(
+                value=None,
+                summary=None,
+                result=dict(
+                    krbmaxticketlife=[u'86400'],
+                    krbmaxrenewableage=[u'604800'],
+                ),
+            ),
+        ),
+
+        dict(
+            desc='Reset user policy',
+            command=(
+                'krbtpolicy_reset', [user1], {}
+            ),
+            expected=dict(
+                value=user1,
+                summary=None,
+                result=dict(
+                    krbmaxticketlife=[u'86400'],
+                    krbmaxrenewableage=[u'604800'],
+                ),
+            ),
+        ),
```

Comment:
I would remove the lines 119 to 147 (Reset global policy and Reset user policy) in order to keep the same strategy throughout the test: define new attr that is incrementally added to the set of policies. Also keep the line krbmaxticketlife=[u'3600'] in the next expected result, after line 156 and in all the subsequent expected results.
You may also use a slightly different value for each maxticketlife to ensure they are not mixed (for instance 3600 for maxticketlife, 3700 for otpmaxticketlife etc...)